### ISSUE TITLE
[DOC]: Add pre-release icon to the README

### ISSUE THREAD

**Issue by Aisuko** [SkywardAI/chat-backend#25]
[![Release Drafter 🚀](https://github.com/SkywardAI/chat-backend/actions/workflows/release-drafter.yml/badge.svg?branch=main)](https://github.com/SkywardAI/chat-backend/actions/workflows/release-drafter.yml)

```
[![Release Drafter 🚀](https://github.com/SkywardAI/chat-backend/actions/workflows/release-drafter.yml/badge.svg?branch=main)](https://github.com/SkywardAI/chat-backend/actions/workflows/release-drafter.yml)
```

Add the code above to the README.md

**Comment by ImranSerifovic** [comment]
Forgot to ask to be assigned but since it was simple went ahead and added it, sent the pull request.

**Comment by Aisuko** [comment]
@ImranSerifovic Thank you. Merged #28
